Open Source/Git1/Ashutosh-Khanduala.txt

```diff
@@ -1,4 +1,4 @@
 Ashutosh Khanduala
 ETC A 2nd Yr
 1902070030
-I love Open source
```

Comment:
I dont know why this line shows like its deleted.

But you can follow those commands to resolve

Reply:
do

```
git reset --soft HEAD~2
git restore --staged Open\ Source/Git1/Ashutosh-Khanduala.txt
git restore Open\ Source/Git1/Ashutosh-Khanduala.txt
```
now make a commit again "task1 completed"
push it

Reply:
Ah I forgot...

to push do `git push -f origin main`